doc/book/examples/sources/basics/transfer.move

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module 0x0::container {
```

Comment:
I think I would call this something other than "container", which evokes a collection type like a vector or map to me. Perhaps `Carrier` (as you use below), `Holder`, or `Wrapper`?